src/ADAL.PCL/TokenCache.cs

```diff
@@ -344,13 +345,16 @@ internal AuthenticationResultEx LoadFromCache(CacheQueryData cacheQueryData, Cal
         {
             lock (cacheLock)
             {
+                CacheEvent cacheEvent = new CacheEvent("cache_lookup");
+                Telemetry.GetInstance().StartEvent(callState.RequestId.ToString(), "cache_lookup");
                 PlatformPlugin.Logger.Verbose(callState, "Looking up cache for a token...");
 
                 AuthenticationResultEx resultEx = null;
                 KeyValuePair<TokenCacheKey, AuthenticationResultEx>? kvp = this.LoadSingleItemFromCache(cacheQueryData, callState);
 
                 if (kvp.HasValue)
                 {
+                    cacheEvent.TokenFound = true.ToString();
                     TokenCacheKey cacheKey = kvp.Value.Key;
```

Comment:
Made a setter for the bool

---
In reply to: [76154674](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/494#discussion_r76154674) [](ancestors = 76154674)